compact-custom-header-editor.js

```diff
@@ -446,10 +480,6 @@ export class CchConfigEditor extends LitElement {
         paper-toggle-button {
           padding-top: 16px;
         }
-        paper-button {
-          background-color: var(--primary-color);

```

Comment:
Without this, the button is transparent and indented 5px like above.

Reply:
Yeah, I figured that buttons shouldn’t get that much attention...